testbed/stack.ts

```diff
@@ -21,12 +21,12 @@ export class Testbed extends cdk.Stack {
                 {
                     name: 'pub-subnet-1',
                     subnetType: ec2.SubnetType.PUBLIC,
-                    cidrMask: 28,
+                    cidrMask: 24,
                 },
                 {
                     name: 'priv-subnet-1',
                     subnetType: ec2.SubnetType.PRIVATE_WITH_NAT,
-                    cidrMask: 28,
+                    cidrMask: 24,
```

Comment:
We should also check that the node group created to have private subnets, currently they get 4 subnets assigned. I am not sure if they get public and private both.

Reply:
It selects only private subnets from VPC for default nodegroups, see here https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/testbed/stack.ts#L69-L70